puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/foreman-packaging-pr-test.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- job:
+    name: foreman-packaging-pr-test
+    project-type: pipeline
+    properties:
+      - github:
+          url: https://github.com/theforeman/foreman-packaging
+    triggers:
+      - github_pr_rpm_develop:
```

Comment:
Can you also set that it only triggers on ```rpm/*``` (unless you start handling debian packaging too)

Reply:
While reviewing I see that you do that in the trigger. Could we use ```github_pr``` and override the white-list-target-branches here inline or am I asking too much?

Reply:
I have no idea how to do that, but patches welcome ;)

Reply:
I think it could be turned into a variable, every job would have to specify '*' though. So there is a tradeoff as far as my understanding of yaml goes.